TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/NoteListFragment.kt

```diff
@@ -270,10 +271,12 @@ class NoteListFragment : Fragment(), AdapterView.OnItemClickListener, AdapterVie
                 true
             }
             R.id.action_select -> {
-                // TODO: toolbar doesn't show up at first, add selection bubbles/visual indication
+                // Select all notes
                 listView?.let {
-                    it.clearChoices()
                     it.choiceMode = ListView.CHOICE_MODE_MULTIPLE_MODAL
+                    for (i in 0 until it.count) {
```

Comment:
I think we did `i in it.indices` before? we should come up with a standard for this kind of loop

Reply:
I tried but it didn't work for ListView :/ I just checked and we were using it in the matching game for an Array